### PR TITLE
Scc 2796

### PR DESCRIPTION
**Pull request by danamansana**
Adds deployments for bibs and items to travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -75,6 +75,70 @@ deploy:
...
+      NYPL_CORE_S3_BASE_URL: https://s3.amazonaws.com
+      PLATFORM_API_BASE_URL: https://qa-platform.nypl.org/api/v0.1/
+      BUCKET_NAME: sierra-poller-state-qa
+      SIERRA_API_BASE_URL: https://nypl-sierra-test.nypl.org:443/iii/sierra-api
+      SIERRA_VERSION: v6
+      SIERRA_OAUTH_URL: https://nypl-sierra-test.nypl.org:443/iii/sierra-api/v3/token
+      SIERRA_OAUTH_ID: AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHoweAYJKoZIhvcNAQcGoGswaQIBADBkBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDA6yJkJXvJKEQUcsCAIBEIA3oerpjzhASuO2ZGudbLoo3dCqgPJfcRu2fj1x2an58I9ldyKNArEv5rzJFYasX3FOrnMeqHdYAA==
+      SIERRA_OAUTH_SECRET: AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHIwcAYJKoZIhvcNAQcGoGMwYQIBADBcBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDLQgrst3U3mqh14f/QIBEIAvR0AVyVvE/sYzsJV2rzFuBslo6xa1nV97F/furZ0V7s7/vZcMQ9gRN1N3B21wDp8=
+      RECORD_TYPE: items
+      RECORD_FIELDS: default,fixedFields,varFields,normTitle,normAuthor,orders,locations
```
Comment (mwbenowitz) [inline .travis.yml:135]: I think this should be different from the `bib` fields, at least as far as I can tell from that other repo? I could definitely be wrong but it looks like it only gets: `default,fixedFields,varFields`

File: .travis.yml [hunk 2]
```diff
@@ -113,6 +177,82 @@ deploy:
...
+  description: A service for polling the Sierra API for updates from the Bibs
+    endpoint
+  region: us-east-1
+  role: arn:aws:iam::946183545209:role/lambda-full-access
+  runtime: ruby2.7
+  timeout: 60
+  module_name: app
+  handler_name: handle_event
+  event:
+    schedule_expression: rate(15 minutes)
```
Comment (mwbenowitz) [inline .travis.yml:191]: Do we know what schedule the current pollers run on? 15 minutes seems reasonable but just want to be sure we keep it the same.

**Review by mwbenowitz — COMMENTED** [review]
This looks good, just want to be sure that the `items` fields are correct.

Also, in the process of doing this we should be sure to document what resources (lambdas, beanstalks and kinesis streams) could be turned off if/when this is deployed. Part of the benefit of this is that it replaces several pieces of infrastructure with a single lambda so we should get that benefit!

File: .travis.yml (hunk 1 above)
Comment (danamansana) [reply .travis.yml:135]: Hmmm... I think you are right. The comment for `requestRangeItem` has all the fields, but the actual code seems to use only three

Comment (mwbenowitz) [reply .travis.yml:135]: yeah, I actually dug into the `sierra-retriever` code now and it's using the `requestMultiItemBasic` method: https://github.com/NYPL-discovery/sierra-wrapper/blob/9fb0f82da0ccab129c6a349d39b355c1e176f658/index.js#L354 and that only uses those three fields.

**Pull request merged**